src/pages/postCreate/components/UploadButton/index.tsx

```diff
@@ -0,0 +1,14 @@
+type SubmitButtonProps = (event: React.MouseEvent) => void;
+
+const UploadButton = ({ onSubmit }: { onSubmit: SubmitButtonProps }) => {
```

Comment:
```suggestion
interface SubmitButtonProps {
  onSubmit: (event: React.MouseEvent) => void;
}
const UploadButton = ({ onSubmit }: SubmitButtonProps) => {
```
현재 코드도 문제는 없지만, 통일성을 주기 위해 props type은 `interface`를 쓰면 어떨까요 ㅎㅎ